### PR TITLE
Scaffold Index missing T usage & Profile Update Issue

### PR DESCRIPTION
**Pull request by macneiln**
Scaffold index missing T usage, erroring when opened.

To recreate: Clone and run py4web locally > open the _scaffold/index route > error occurs due to missing T import.

### CODE REVIEW

**Comment by macneiln** [comment]
Commit [7118e0d](https://github.com/web2py/py4web/pull/696/commits/7118e0d2ea11f776c943eba0543b8171dad27709) fixes issue #697, as passing the table fields directly into the Form class sets the dbio flag to false if the value passed to the class is a list rather than the actual table object.

**Comment by macneiln** [comment]
Commit [7118e0d](https://github.com/web2py/py4web/pull/696/commits/7118e0d2ea11f776c943eba0543b8171dad27709) fixes issue #697, as passing the table fields directly into the Form class sets the dbio flag to false if the value passed to the class is a list rather than the actual table object.

File: py4web/utils/auth.py [hunk 1]
```diff
@@ -1664,7 +1664,7 @@ def profile(self, model=False):
                         deletable=deletable)
 
         form = Form(
-            fields,
+            self.auth.db.auth_user,
```
Comment (mdipierro) [inline py4web/utils/auth.py:1667]: why this change?

Comment (macneiln) [reply py4web/utils/auth.py:1667]: Hi Massimo, this was something I broke with my previous Auth API updates.

The commit fixes issue #697 where the profile details cannot be updated currently due to behaviour in the Form class setting the dbio flag to false if the value passed to the class is a list object rather than the actual table object.
